sassc.c

```diff
@@ -202,6 +202,7 @@ void print_usage(char* argv0) {
     printf("   -m, --sourcemap         Emit source map.\n");
     printf("   -M, --omit-map-comment  Omits the source map url comment.\n");
     printf("   -p, --precision         Set the precision for numbers.\n");
+    printf("   -a, --sass              Treat input as indented syntax.\n");
```

Comment:
I agree with @mgreter. Indented syntax is largely internal terminology, people think of it as sass and scss syntax.

I also agree that the short syntax is probably not useful, but it's not harmful so I don't feel strongly.